OOP/Assignment 1/OOPQuestion1/MatrixProgramMain.java

```diff
@@ -0,0 +1,42 @@
+package OOPQuestion1;
+
+import java.util.Scanner;
+
+public class MatrixProgramMain
+{
+	public static void main(String args[])
+	{
+		Scanner sc = new Scanner(System.in);
+		int row, col;
+		System.out.println("Enter the number of rows and columns of matrix");
+		row = sc.nextInt();
+		col = sc.nextInt();
+		int[][] array = new int[row][col];
+		Matrix mat = new Matrix(row, col);
+
+		System.out.println("Enter the elements of matrix");
+		for (int i = 0; i < row; i++)
+		{
+			for (int j = 0; j < col; j++)
+			{
+				int value = sc.nextInt();
+				array[i][j] = value;
+			}
+		}
+		mat.addElement(array);
+		sc.close();
+		mat.tanspose();
```

Comment:
i am changing the matrix one i call transpose method... no need to assign it again.. method still returning altered object... it will consume one extra line...so i didnt catch it